src/styles/objects.Course.scss

```diff
@@ -1,44 +1,58 @@
+@import 'settings.constants.scss';
+
 .course {
+  list-style: none outside none;
+  padding: 0;
+  text-align: center;
+  width: $course-box-width;
+  height: $course-box-height;
+  color: $course-box-font-color;
+  margin-bottom: 10px;
+
   .wrapper {
     display: flex;
     flex-flow: column;
     overflow: hidden;
     height: 100%;
     background: inherit;
-    padding: 0 5px 5px 5px;
   }
 
   .header-text {
     font-weight: bolder;
     font-size: x-small;
     padding: 1px;
     margin: 0;
-    margin-top: 0px;
+    margin-top: -14.26px;
```

Comment:
...oh...
.26px?
...
carry on

Reply:
Yeah, I shouldn't drink when writing CSS, this is what I end up doing 😆